templates/events/_header.html.twig

```diff
@@ -35,7 +35,11 @@
                 L'événement à été annulé
             </div>
         {% elseif not event.full%}
-            <a class="btn btn--no-border {{ isFull?' btn--full':''}} register-event" href="{{ path("app_#{event.type}_attend", {slug: event.slug}) }}">Je veux participer</a>
+            {% if is_already_participating(event, app.user) %}
+                <button class="btn btn--no-border {{ isFull?' btn--full':''}} unregister-event" data-csrf-token="{{ csrf_token('citizen_action.unregistration') }}">Je ne veux plus participer</button>
```

Comment:
I think it's the wrong place for this button, because we are in `elseif not event.full`, when the event will be full I'll not see this button.